spec/requests/unlock_account_spec.rb

```diff
@@ -0,0 +1,74 @@
+RSpec.describe "Unlock account" do
+  include ActiveJob::TestHelper
```

Comment:
Do you need the ActiveJob test helper for a request spec?
I wonder if its included already in the spec config..? Will go look.

Reply:
The test failed without this, as `assert_enqueued_jobs` was undefined

Reply:
We could possibly include it in the spec_helper.rb file (as a `config.include`)

Reply:
Ah couldn't find anything, I thougt for a second it might be smuggled in with some devise stuff.
But if it falls over when you remove it then no prob.

I don't think we should include it in the test helper unless it's turning up everywhere!
Thanks again will approve.